burn-compute/Cargo.toml

```diff
@@ -17,19 +17,33 @@ default = [
     "channel-mpsc",
     "channel-cell",
     "storage-bytes",
+    "autotune-persistent-cache",
+]
+std = [
+    "burn-common/std",
 ]
-std = ["burn-common/std"]
 channel-mutex = []
 channel-cell = []
 channel-mpsc = [] # Assume std
 storage-bytes = []
+autotune-persistent-cache = [
+    "dirs",
+    "md5",
+    "serde",
+    "serde_json",
+] # Assume std
 
 [dependencies]
 burn-common = { path = "../burn-common", version = "0.12.0", default-features = false }
 derive-new = { workspace = true }
 spin = { workspace = true }
 log = { workspace = true }
 hashbrown = { workspace = true }
+dirs = { workspace = true , optional = true}
+serde = { workspace = true, optional = true}
+serde_json = { workspace = true, features=["std"], optional = true}
+md5 = { version = "0.7.0", optional = true }
+rand.workspace = true
```

Comment:
Should it be optional or in the dev-dependencies section (I think you are only using it in tests)?

Reply:
Moved to dev deps.